### PR TITLE
Upgrade obytelib to version 1.5, ocamlclean to version 2.2 and ocapic to version 3.4.

### PR DESCRIPTION
**Pull request by bvaugon**

### CODE REVIEW

**Comment by bvaugon** [comment]
Build failed on MacOS because of the **graphics** library which seems to do not compile on MacOS. Since the problem comes from graphics and not from the commited packages, and only on MacOS, I think this pull request may be merged.

File: packages/ocapic/ocapic.3.4/opam [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+]
+remove: [
+  ["./configure" "-prefix" prefix]
+  [make "uninstall"]
+]
+depends: [
+  "ocaml" {>= "4.07.0"}
+  "camlp4"
+  "graphics"
+  "ocamlbuild"
```
Comment (kit-ty-kate) [inline packages/ocapic/ocapic.3.4/opam]:
```suggestion
  "ocamlbuild" {build}
```
do you need ocamlbuild at runtime?

File: packages/ocamlclean/ocamlclean.2.2/opam [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  ["./configure" "-prefix" prefix]
+  [make "all"]
+]
+remove: [
+  ["./configure" "-prefix" prefix]
+  [make "uninstall"]
+]
+depends: [
+  "ocaml" {>= "4.07.0"}
+  "ocamlbuild"
```
Comment (kit-ty-kate) [inline packages/ocamlclean/ocamlclean.2.2/opam]: same comment here

**Comment by kit-ty-kate** [comment]
otherwise apart from my two comments, it looks indeed fine to merge

File: packages/ocamlclean/ocamlclean.2.2/opam (hunk 2 above)
Comment (bvaugon) [reply packages/ocamlclean/ocamlclean.2.2/opam]: Yes, indeed. Thanks!

**Comment by mseri** [comment]
Thanks!

**Pull request merged**